sos/plugins/openvswitch.py

```diff
@@ -42,7 +42,10 @@ def setup(self):
             # Gather the database.
             "ovsdb-client -f list dump",
             # List the contents of runtime directory
-            "ls -laZ /var/run/openvswitch"
+            "ls -laZ /var/run/openvswitch",
+            # Gather systemd services logs
+            "journalctl -u openvswitch",
+            "journalctl -u openvswitch-nonetwork"
```

Comment:
(we may want to revisit this as we're getting some reports of problems with journald logs for services that write out huge quantities of data - for now though this is consistent with the other journalctl unit usage in sos).